patches/store-socket.diff

```diff
@@ -24,14 +24,36 @@ Index: code-server/lib/vscode/src/vs/workbench/api/node/extHostExtensionService.
  import * as performance from 'vs/base/common/performance';
  import { createApiFactoryAndRegisterActors } from 'vs/workbench/api/common/extHost.api.impl';
  import { RequireInterceptor } from 'vs/workbench/api/common/extHostRequireInterceptor';
-@@ -72,6 +74,10 @@ export class ExtHostExtensionService ext
- 		if (this._initData.remote.isRemote && this._initData.remote.authority) {
- 			const cliServer = this._instaService.createInstance(CLIServer);
- 			process.env['VSCODE_IPC_HOOK_CLI'] = cliServer.ipcHandlePath;
-+
-+			fs.writeFile(path.join(os.tmpdir(), 'vscode-ipc'), cliServer.ipcHandlePath).catch((error) => {
```

Comment:
I had to move this below "Module loading tricks" in order for the workspace to show up with Folders. Not familiar enough with the VSCode codebase to know why.

Reply:
I will check it tomorrow!

Reply:
Hmm yeah mine are still empty for some reason!  I tried both a workspace file (output listed above) and opening a folder directly.  That one looks like this:

```
{"workspace":{"id":"6b5587dc","folders":[],"transient":false,"configuration":null},"socketPath":"/run/user/1002/vscode-ipc-01935aa6-3ef8-4a7c-8f7b-732ec465af53.sock"}
```